src/nyc_trees/sass/partials/_buttons.scss

```diff
@@ -5,6 +5,55 @@
   }
 }
 
+.btn-group-2, .btn-group-3, .btn-group-4 {
+  vertical-align: middle;
+  display: table;
+  width: 100%;
+  table-layout: fixed;
+  border-collapse: separate;
+  min-height: 45px;
+
+  .btn {
+    display: table-cell;
+    float: none;
+    vertical-align: middle;
+    padding: 8px;
+
+    // This fixes the white space between buttons caused by the checkbox/radio btns
+    // Bad Practice
+    // Look into a better solution for this fix
```

Comment:
Since we missed the boat on getting this fixed in time for user testing, it might be best to clean this up now.